projectile.el

```diff
@@ -4493,6 +4493,16 @@ directory to open."
   (make-hash-table :test 'equal)
   "A mapping between projects and the last run command used on them.")
 
+(defvar projectile-project-enable-cmd-caching t
+  "Enables command caching for the project.  Set to nil to disable.
+Should be set via .dir-locals.el.")
+
+(defun projectile--cache-project-commands-p ()
```

Comment:
Is this really necessary? Seems to me that the local variable would work fine in most cases even without it.

Reply:
my understanding is that it will fail in the case where the f.ex. *compilation* buffer is selected. this
is not too uncommon for me personally, and noticed it quite fast with my initial attempt which did
not do this.

it's not necessary, but it's more ergonomic. up to you.

Reply:
Okay, fair enough. We can keep this.